Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/authorization/TryCreateBootstrapAuthorizationAdmin.ts

```diff
@@ -44,7 +44,14 @@ export async function tryCreateBootstrapAuthorizationAdminDocument(
           'mongoCollection.replaceOne returned acknowledged: false, indicating a problem with write concern configuration';
         Logger.error(functionName, request.traceId, msg);
       } else {
-        Logger.debug(`${functionName}: Inserted ${upsertedCount} documents`, request.traceId);
+        if (upsertedCount == 1) {
```

Comment:
`if (upsertedCount === 1) {`

Reply:
Not sure if the double equal comparison is intentional since I see other parts of the code where the verification is done with loose equality.

Reply:
Should be triple equal - I commonly miss that when switching between Python (or C#) and TypeScript. Linter should be failing too.

Reply:
```suggestion
        if (upsertedCount === 1) {
```